core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -240,6 +248,16 @@ object TiUtil {
           conf.setKeyCertChainFile(sqlConf.getConfString(TiConfigConst.TIKV_KEY_CERT_CHAIN))
           conf.setKeyFile(sqlConf.getConfString(TiConfigConst.TIKV_KEY_FILE))
         }
+
```

Comment:
can  we use sparkConfToTiConfWithoutPD to create TiConf in TiCatalog? so that we needn't modify two places

Reply:
TiContext is inited before TiCatalog, and TiSession will be cached  after the init of  TiContext. Thus, TiCatalog just need pd address to get TiSession which is cached.
As for the TLS config, I think it has been added into the configuration when TiSession is created in TiContext. injectTLSParam is redundant.

Reply:
@shiyuhang0 Great. You are right. I have deleted the duplicate, redundant, unuseful function injectTLSParam().